optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/viewport/rotation/ShiftTemplateGridObject.java

```diff
@@ -59,6 +73,12 @@ public void setEndPositionInScaleUnits(LocalDateTime newEndPosition) {
     @Override
     public void init(Lane<LocalDateTime, RotationMetadata> lane) {
         updatePositionInLane();
+        if (shiftTemplateView != null) {
+        label.innerHTML =  (shiftTemplateView.getRotationEmployeeId() != null)? new SafeHtmlBuilder().appendEscaped(getLane().getMetadata()
```

Comment:
@Christopher-Chianelli Please keep the code properly formatted (indentation, extra/missing spaces).

Also, please try to avoid lines longer than 128 characters when possible - it will make reading the code easier.

Reply:
Done. The eclipse formatter for KIE automatically indents chained methods calls to line-up, which often causes long chain of methods to be pushed very far (causing lines to be much longer than need-be).

Reply:
Judiciously placed line break will solve that :-)